script.js

```diff
@@ -221,4 +221,38 @@ feature6.addEventListener('click', () => {
 
 IconClose6.addEventListener('click', () => {
   popup6.classList.toggle('active');
-});
\ No newline at end of file
+});
+
+const contactForm = document.getElementById('contact-form');
+const fullName = document.getElementById('name');
+const email = document.getElementById('email');
+const message = document.getElementById('message');
+const errorMessage = document.getElementById('error-message');
+
+function formValidate(event) {
```

Comment:
- [ ] Please 🙏🏻 adhere to the best practices of JavaScript and use modern syntax for your code because it will make sure that you are following the current industries standards and writing good quality code, right now you are using ES5 javascript code, but please use the **ES-Next** syntax according to which you need to use the `arrow` function. Kindly refactor your code according to latest standards.

Reply:
@skyv26 the MIT wad added that means you review not professional? when you check files you need to know you take chance form other person.